icekit/bin/setup-postgres.sh

```diff
@@ -10,6 +10,13 @@ EOF
 
 set -e
 
+# Is the `pv` utility available
+if [[ `pv --help > /dev/null 2>&1` ]]; then
+    PV_CMD=pv
+else
+    PV_CMD=cat
+fi
```

Comment:
The `alias pv=cat` trick was a good idea but didn't work. I've kept the `$PV_CMD` variable for now